api/src/main/java/com/cloud/vm/VmDetailConstants.java

```diff
@@ -53,6 +53,9 @@ public interface VmDetailConstants {
     String NIC_MULTIQUEUE_NUMBER = "nic.multiqueue.number";
     String NIC_PACKED_VIRTQUEUES_ENABLED = "nic.packed.virtqueues.enabled";
 
+    // KVM specific, disk controllers
+    String KVM_SKIP_FORCE_DISK_CONTROLLER = "skip.force.disk.controller";
```

Comment:
@shwstppr should it be included for for KVM here: https://github.com/apache/cloudstack/blob/e90e436ef873100b1916c80c0309c1eefcc4a265/server/src/main/java/com/cloud/api/query/QueryManagerImpl.java#L5060-L5061

also, check if any doc changes needed for this vm detail.